src/components/ContentHighlights/HighlightStepper/ContentConfirmContentCard.jsx

```diff
@@ -0,0 +1,79 @@
+import React, { useEffect, useState } from 'react';
+import PropTypes from 'prop-types';
+import { Delete } from '@edx/paragon/icons';
+import { IconButton, Icon } from '@edx/paragon';
+import { connect } from 'react-redux';
+import ContentHighlightCardItem from '../ContentHighlightCardItem';
+import { useContentHighlightsContext } from '../data/hooks';
+import { generateAboutPageUrl } from '../data/constants';
+
+const ContentConfirmContentCard = ({ enterpriseSlug, original }) => {
+  const { deleteSelectedRowId } = useContentHighlightsContext();
+  const [deleteKey, setDeleteKey] = useState(null);
+  const [isDeleted, setIsDeleted] = useState(false);
+  const {
+    title,
+    contentType,
+    partners,
+    cardImageUrl,
+    originalImageUrl,
+    firstEnrollablePaidSeatPrice,
+    aggregationKey,
+    key,
+  } = original;
+  // eslint-disable-next-line no-shadow
+  useEffect(() => {
+    if (deleteKey === aggregationKey) {
+      deleteSelectedRowId(deleteKey);
+      setIsDeleted(true);
+    }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [deleteKey, aggregationKey]);
+  if (isDeleted) {
+    return (
+      null
+    );
+  }
+  return (
+    <React.Fragment key={key}>
+      <ContentHighlightCardItem
+        title={title}
+        hyperlink={generateAboutPageUrl(enterpriseSlug, contentType?.toLowerCase(), aggregationKey?.split(':')[1])}
+        contentType={contentType}
+        partners={partners}
+        cardImageUrl={cardImageUrl || originalImageUrl}
+        price={firstEnrollablePaidSeatPrice}
+
+      />
+      <IconButton
+        invertColors
+        isActive
+        src={Delete}
+        iconAs={Icon}
+        alt="Delete"
```

Comment:
nit: a bit of an improvement in the `alt` text here might be something like, "`Remove ${title} from highlight collection`" so that each content card has a more explicit, unique `alt`.